src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * Represents a Tag in the address book.
+ * Guarantees: immutable; name is valid as declared in {@link #isValidTagName(String)}
+ */
+public class Tagging {
+	public Person person;
+	public Tag tag;
+	public String status;
```

Comment:
think why `String` may not be the best data type for this particular field?
what would be more appropriate?